mod/display.php

```diff
@@ -43,71 +43,61 @@ function display_init(App $a)
 
 	$r = false;
 
+	$fields = ['id', 'parent', 'author-id', 'body', 'uid'];
+
 	// If there is only one parameter, then check if this parameter could be a guid
 	if ($a->argc == 2) {
 		$nick = "";
 		$r = false;
```

Comment:
Code clarity: Please rename `$r` to `$item` in the whole function. Additionally, please change the variable initialization to `$item = null;`.